### PR TITLE
当手动重跑历史活着补数的时候对shell和sql脚本中的[YYYYmmddd...]变量赋值与传递的日期，而不是当前日期

### PR DESCRIPTION
**Pull request by rusonding**
## *Tips*
- *Thanks very much for contributing to Apache DolphinScheduler.*
- *Please review https://dolphinscheduler.apache.org/en-us/community/index.html before opening a pull request.*

## What is the purpose of the pull request

*(For example: This pull request adds checkstyle plugin.)*

## Brief change log

*(for example:)*
  - *Add maven-checkstyle-plugin to root pom.xml*

## Verify this pull request

*(Please pick either of the following options)*

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

*(example:)*

  - *Added dolphinscheduler-dao tests for end-to-end.*
  - *Added CronUtilsTest to verify the change.*
  - *Manually verified the change by testing locally.*

### CODE REVIEW

**Comment by rusonding** [comment]
历史重跑或者补数的时候，替换传递的日期（也就是数据日期），而不是当前日期

File: dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/worker/task/sql/SqlTask.java [hunk 1]
```diff
@@ -236,6 +236,9 @@ private SqlBinds getSqlAndSqlParamsMap(String sql) {
             sqlParameters.setTitle(title);
         }
 
+        //new add,手动执行，手动补数，自动调度，历史重跑调度日期替换$[yyyy-MM-dd]
```
Comment (dailidong) [inline dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/worker/task/sql/SqlTask.java]: English is needed

File: dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/ParameterUtils.java [hunk 2]
```diff
@@ -78,6 +78,68 @@ public static String convertParameterPlaceholders(String parameterString, Map<St
     return parameterString;
   }
 
+  /**
+   * new
+   * 新增方法替换调度时间
```
Comment (dailidong) [inline dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/ParameterUtils.java]: English is needed

**Comment by dailidong** [comment]
any update？

**Pull request closed without merging**